### DISCUSSION TITLE
Rename branches

### DISCUSSION THREAD

**Discussion by haowang-bioinfo** [SysBioChalmers/Human-GEM#226] [Ideas]
I think this should be a general enough topic for discussion: should we just adapt from **_devel_**  to  **_develop_** branch in our workflow?

The use of _develop_ was suggested by `standard-GEM` [v0.5](https://github.com/MetabolicAtlas/standard-GEM/blob/master/.standard-GEM.md). Now (right after Human v1.6) probably is a good time point to make this change.

**Comment by JonathanRob** [comment]
Personally I prefer `dev`, but I realize that `develop` is more descriptive, and therefore probably a better choice.

**Comment by mihai-sysbio** [comment]
> The use of _develop_ was suggested by `standard-GEM` [v0.5](https://github.com/MetabolicAtlas/standard-GEM/blob/master/.standard-GEM.md).

The use of _develop_ is actually coming from the `git flow` best practices. It should have been documented in `standard-GEM` that this was not arbitrarily decided.

**Comment by haowang-bioinfo** [comment]
With the consensus, a new branch `develop` was created from, and will replace, the current `devel` branch.

**Reply by mihai-sysbio** [reply]
Another way to do achieve the same result is to rename the branch locally, and push, followed by a deletion of the original branch.

**Comment by haowang-bioinfo** [comment]
[GitHub](https://github.com/github/renaming) has been actively working on renaming the default branch of repositories from `master` to `main`. Now it seems straightforward to make this change from the [web](https://github.com/github/renaming#renaming-existing-branches). I thus propose such a change to Human-GEM repo asap.

**Reply by mihai-sysbio** [reply]
Great idea!

**Reply by mihai-sysbio** [reply]
Would you prefer that I do this?

**Reply by haowang-bioinfo** [reply]
yes, go ahead

**Reply by mihai-sysbio** [reply]
It is done.

**Reply by JonathanRob** [reply]
Thanks for making this change, @mihai-sysbio. We will still need to update other occurrences of `master`; e.g., in the README.

**Reply by mihai-sysbio** [reply]
Oh that totally slipped my mind, thanks for mentioning it. Here is the PR https://github.com/SysBioChalmers/Human-GEM/pull/284.
